env/work/srw_python/srwlib.py

```diff
@@ -4584,8 +4614,7 @@ def srwl_wfr_emit_prop_multi_e(_e_beam, _mag, _mesh, _sr_meth, _sr_rel_prec, _n_
 
             #MR20160907 #Save .log and .json files:
             particle_number = (i + 1) * _n_part_avg_proc
-            #srwl_save_status(particle_number, total_num_of_particles)
-            srwl_uti_save_stat_wfr_emit_prop_multi_e(particle_number, total_num_of_particles)
+            srwl_uti_save_stat_wfr_emit_prop_multi_e(particle_number, total_num_of_particles, filename=log_path)
```

Comment:
I'll port these parts (change calls to srwl_uti_save_stat_wfr_emit_prop_multi_e) after the previous comment is addressed, to ensure consistency.